main.py

```diff
@@ -2,6 +2,25 @@
 
 from django.db.models import QuerySet
 
+from db.models import Genre, Actor
+
 
 def main() -> QuerySet:
-    pass
+    genres = ["Western", "Action", "Dramma"]
+    for genre in genres:
+        Genre.objects.get_or_create(name=genre)
+    actors = [("George", "Klooney"),
+              ("Kianu", "Reaves"),
+              ("Scarlett", "Keegan"),
+              ("Will", "Smith"),
+              ("Jaden", "Smith"),
+              ("Scarlett", "Johansson")]
+    for first_name, last_name in actors:
+        Actor.objects.create(first_name=first_name, last_name=last_name)
+    Genre.objects.filter(name="Dramma").update(name="Drama")
+    Actor.objects.filter(first_name="George").update(last_name="Clooney")
+    (Actor.objects.filter(first_name="Kianu")
+     .update(first_name="Keanu", last_name="Reeves"))
+    Genre.objects.filter(name="Action").delete()
+    Actor.objects.filter(first_name="Scarlett").delete()
```

Comment:
Deleting all actors with the first name 'Scarlett' will remove both 'Scarlett Keegan' and 'Scarlett Johansson'. Ensure this is the intended behavior, as it might be too broad.